cpp/kiss_icp/pipeline/KissICP.hpp

```diff
@@ -74,6 +75,7 @@ class KissICP {
     std::vector<Eigen::Vector3d> LocalMap() const { return local_map_.Pointcloud(); };
     Sophus::SE3d pose() const { return last_pose_; }
     Sophus::SE3d delta() const { return last_delta_; }
+    const KISSConfig &config() const { return config_; }
```

Comment:
This is fine :+1:

Reply:
Okay, but without the empty constructed it is not needed, because then you need to provide a config anyway to initialize a pipeline :(